README.md

```diff
@@ -54,19 +49,28 @@ for more background, objectives, key use cases.
 
 2. Navigate to [localhost:6060/](http://localhost:6060/)
 
-#### Test Data
+#### Database
+
+To create the database tables ...
+
+1. Create the `blurts` database:
 
-To create the test database tables, run the `knex` migrations:
+   ```sh
+   createdb blurts
+   ```
 
-* `./node_modules/.bin/knex --knexfile db/knexfile.js migrate:latest`
+2. Update the `DATABASE_URL` value in your `.env` file with your local db
+   credentials:
 
-To populate the database with test data, use these scripts in `scripts/`:
+   ```
+   DATABASE_URL="postgres://<username>@localhost:<port>/blurts"
+   ```
 
-* `node scripts/load-breaches.js` loads breaches from HIBP into the database
-* `node scripts/add-breached-emails.js` adds some breached email records for
-  `test[1-3]@test.com` users
+3. Run the migrations:
 
-* `node scripts/add-breached-emails.js --help` for usage help.
+   ```
+   ./node_modules/.bin/knex --knexfile db/knexfile.js migrate:latest
```

Comment:
nit: Not sure if it's worth adding these to an npm script to avoid having to use the `./node_modules/bin/` prefix, and we could just run <kbd>$ npm run db:migrate</kbd>.
Where package.json has the following script:

```json
"db:migrate": "knex --knexfile db/knexfile.js migrate:latest"
```